curriculum/challenges/english/10-coding-interview-prep/rosetta-code/compare-a-list-of-strings.md

```diff
@@ -8,13 +8,13 @@ dashedName: compare-a-list-of-strings
 
 # --description--
 
-A list or sequence is an abstract data type that represents a finite number of ordered values, where the same value may occur more than once. Here is an example:
+A list is a type of data that shows values, you may notice its similarity to sequence from other Rosetta challenges, which is true. As they can have a repeating values, here is an example:
 
 ```js
 const list = [['AA',  'BB', 'CC'], ['AA', 'ACB', 'AA'], [], ['AA']];
 ````
 
-Given a list of arbitrarily many strings, implement a function for each of the following conditions:
+Given a <a href="https://rosettacode.org/wiki/Compare_a_list_of_strings" target="_blank" rel="noopener noreferrer nofollow">list</a> of arbitrarily many strings, implement a function for each of the following conditions:
```

Comment:
Why was a link added here? I'm not sure it's needed? I thought the plan was to link once in the Superblock description?